app/ui/src/main/java/com/fsck/k9/fragment/MessageListAdapter.kt

```diff
@@ -62,6 +63,9 @@ class MessageListAdapter internal constructor(
         array.recycle()
     }
 
+    private inline val previewLines: Int
```

Comment:
Instead of moving around the code that is accessing settings via `K9` I'd prefer to pass in the settings. It's probably best to group all of these settings in a `MessageListAppearance` data class.

Reply:
Yep, thought about this during last commute. I'll try to pack it up. Down the line this could allow for adapter refresh on settings change.